internal/worker.go

```diff
@@ -88,6 +88,7 @@ type (
 		// rate at which the worker is able to consume tasks from a task list,
 		// unless FeatureFlags.PollerAutoScalerEnabled is set to true.
 		// Default value is 1
+		// Deprecated: Use AutoScalerOptions instead.
```

Comment:
probably worth calling out that these are also completely ignored and do nothing, not just the ones below.

and maybe [something] on the max value, since it's now just "the poller count" if not using the autoscaler?